docs/integrate/tableau/tutorial.md

```diff
@@ -0,0 +1,70 @@
+(tableau-tutorial)=
+# Using CrateDB with Tableau
+
+For using Tableau with CrateDB, install the latest PostgreSQL driver, as detailed in the steps below.
+
+1. Download the Java 8 JDBC driver (e.g. `postgresql-42.7.1.jar` or newer) from https://jdbc.postgresql.org/download/. 
+2. Move the downloaded `.jar`-file (e.g. `postgresql-42.7.1.jar`) into the Tableau Driver folder.
+    > * **Windows**: `C:\Program Files\Tableau\Drivers`
+    > * **Mac**: `~/Library/Tableau/Drivers`
+    > * **Linux**: `/opt/tableau/tableau_driver/jdbc`
+3. Open Tableau.
+4. Create a `PostgreSQL` connection.
+
+   ![image|454x500, 75%](https://us1.discourse-cdn.com/flex020/uploads/crate/original/2X/c/cf27bb4288737b66d2af620092f529f481dbe328.jpeg){height=500px}
+5. Start using Tableau with CrateDB.
```

Comment:
@surister: That's the original from [Using CrateDB with Tableau](https://community.cratedb.com/t/using-cratedb-with-tableau/1192). It should probably be replaced or refer to the [new connector's "Install" section](https://github.com/crate/cratedb-tableau-connector?tab=readme-ov-file#install) instead?